From f9c6bd5aa6918412a32bfb803ff0a1a68646b0e2 Mon Sep 17 00:00:00 2001
From: Alain Volmat <alain.volmat@foss.st.com>
Date: Wed, 2 Oct 2024 11:38:01 +0200
Subject: [PATCH 6/7] fixup! media: dcmipp: statcap: stat region & pixel count
 rework

Correct a typo leading to a wrong height value of region being set
when there is no region selected.

Change-Id: I121b104c047865a082e220b8cd23dc5c1a3670fb
Signed-off-by: Alain Volmat <alain.volmat@foss.st.com>
---
 drivers/media/platform/st/stm32/stm32-dcmipp/dcmipp-statcap.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/st/stm32/stm32-dcmipp/dcmipp-statcap.c b/drivers/media/platform/st/stm32/stm32-dcmipp/dcmipp-statcap.c
index 85fe5d4036c3..24953b95f3ca 100644
--- a/drivers/media/platform/st/stm32/stm32-dcmipp/dcmipp-statcap.c
+++ b/drivers/media/platform/st/stm32/stm32-dcmipp/dcmipp-statcap.c
@@ -557,7 +557,7 @@ static int dcmipp_statcap_s_ctrl(struct v4l2_ctrl *ctrl)
 		     !region->left[0] && !region->top[0])) {
 			spin_lock_irq(&vcap->irqlock);
 			vcap->stat_region.width = sel.r.width;
-			vcap->stat_region.height = sel.r.width;
+			vcap->stat_region.height = sel.r.height;
 			reg_clear(vcap, DCMIPP_P1STSZR, DCMIPP_P1STSZR_ENABLE);
 			spin_unlock_irq(&vcap->irqlock);
 			break;
-- 
2.25.1

